bin/validateSchema.ts

```diff
@@ -92,28 +84,27 @@ const REGION_MAP = {
   ap: Region.AP,
 } as const;
 
-// Update event request is not yet supported in the Node SDK
-type UpdateEventArgs = {
-  requestId: string;
-  subscription: TestSubscription;
-  payload: any;
-};
-
-function updateEventRequest({ requestId, subscription, payload }: UpdateEventArgs) {
-  const regionPrefix = subscription.region === 'us' ? '' : `${subscription.region}.`;
-  return fetch(`https://${regionPrefix}api.fpjs.io/events/${requestId}`, {
-    method: 'PUT',
-    headers: { 'Content-Type': 'application/json', 'Auth-API-Key': subscription.serverApiKey },
-    body: JSON.stringify(payload),
-  });
+function getCurrentFunctionName() {
+  try {
+    throw new Error();
+  } catch (e) {
+    // Parse the stack trace to get the function name
+    const stackLines = e.stack.split('\n');
+    // The calling function is typically at index 2 (after Error and getCurrentFunctionName)
+    const callerLine = stackLines[2];
+    // Extract the function name using regex
+    const match = callerLine.match(/at\s+(.*)\s+\(/);
+    return match ? `${match[1]}()` : 'anonymous';
+  }
```

Comment:
This is an interesting approach, but it feels a bit "hacky".
Wouldn't it be easier to just provide the function name when logging, instead of calling this? I imagine the function names don't change that often to justify it, but I'll leave it up to you.

Reply:
Hacky in a good way? O:) Jk, you are right, refactored: https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/200/commits/42bd7aa3162c19e170fd0c9b4ec353c75e93ea1f